src/timedate/timedatectl.c

```diff
@@ -261,6 +261,15 @@ static int set_local_rtc(int argc, char **argv, void *userdata) {
         if (b < 0)
                 return log_error_errno(b, "Failed to parse local RTC setting '%s': %m", argv[1]);
 
+        if (b == 1) {
+                fflush(stdout);
+                log_warning("Warning: The system is now being configured to read the RTC time in the local time zone\n"
+                       "         This mode cannot be fully supported. It will create various problems\n"
+                       "         with time zone changes and daylight saving time adjustments. The RTC\n"
+                       "         time is never updated, it relies on external facilities to maintain it.\n"
+                       "         If at all possible, use RTC in UTC");
```

Comment:
hmm, indentation is a bit strange now. please indent thins so that these continuation lines are aligned to the first line of the warning.

Reply:
(i.e. indent 6 spaces to the right)

Reply:
It actually looks correct after compilation btw:
```
$ ./timedatectl set-local-rtc 1
Warning: The system is now being configured to read the RTC time in the local time zone
         This mode cannot be fully supported. It will create various problems
         with time zone changes and daylight saving time adjustments. The RTC
         time is never updated, it relies on external facilities to maintain it.
         If at all possible, use RTC in UTC

$ ./timedatectl
               Local time: Tue 2024-07-09 21:36:25 IST
           Universal time: Tue 2024-07-09 16:06:25 UTC
                 RTC time: Tue 2024-07-09 21:36:25
                Time zone: Asia/Kolkata (IST, +0530)
System clock synchronized: yes
              NTP service: active
          RTC in local TZ: yes

Warning: The system is configured to read the RTC time in the local time zone.
         This mode cannot be fully supported. It will create various problems
         with time zone changes and daylight saving time adjustments. The RTC
         time is never updated, it relies on external facilities to maintain it.
         If at all possible, use RTC in UTC by calling
         'timedatectl set-local-rtc 0'.
```
Actual warning message is tab'ed against the "Warning:" label

Reply:
(Please align the quotes in the source too.)

Reply:
Oh got your point. Added new commit. Sorry for negligence